src/common/components/FontAwesomeSvgIcon.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function FontAwesomeSvgIcon({fontAwesomeIcon: {width, height, svgPathData}, ...rest}) {
```

Comment:
nit: `...props`

Reply:
`...restProps` as that's whats used elsewhere